ROADMAP.md

```diff
@@ -1,25 +1,389 @@
+* Kick off fundraising campaign to provide multi-year funding for ongoing operations, SRE, infrastructure costs
+* Frees up community resources to focus on development & standardization
+
+### **Pursue open standards-track ratification of APIs, signature attestation formats via IETF**
+
+* Leverage documents written for ISRG as baseline RFC proposals
+
+### **Ensure Rekor and Fulcio are sufficiently robust & scalable for anticipated growth**
+
+* Complete performance analysis to ensure appropriate rate limits and incremental cost vectors are known
+* Perform regular sharding and key rotation actions to ensure freshness and accuracy of playbooks
+
+### **Make Sigstore infrastructure services simpler to operate for both public and private operators**
+
+* Continue to invest in infrastructure tooling (e.g. Helm charts, Terraform modules) that simplify deployment and management of Sigstore services
+* Continue to collaborate with TUF ecosystem to make parallel and delegated roots of trust fully supported
+
+### **Create SIGs within Sigstore community to drive focused discussion & execution around critical topics**
+
+* SIG-clients to foster collaboration and standardization between Sigstore clients
+* SIG-ops to focus on the operation of the public-good infrastructure and handoff to a neutral entity
+
+### **Create excellent onboarding path for users and log operators across both technical and non-technical dimensions**
+
+* Explain what a log does (and does not) provide
+
+### **Work with the transparency, academic, and package repositories communities to monitor log consistency**
+
+* Deploy log witnesses across different trust boundaries
+* Encourage community members to operate log witnesses
+* Develop a witness network with a tiered approach to log witnesses and distributors
+* Provide tooling for individuals to monitor logs for their signing identities
+
+### **Collaborate with academic community to provide stronger trust assurances and privacy guarantees**
+
+* Provide attestations of trusted actions, such as verifying identity tokens, to minimize needed trust in service operators
+* Use modern cryptography (zero-knowledge proofs, verifiable random functions) to remove PII where possible
+
+### **Ensure that verification is as seamless as signing**
+
+* Integrate with existing verification policy languages (e.g. Rego, CUE)
+* Provide a reference implementation for how to verify signatures across various points in the software development lifecycle
+
+### **Offer intuitive and ergonomic tooling for engineers and cloud operations**
+
+* [Cosign CLI](https://github.com/sigstore/cosign) is the official Sigstore CLI multi-tool for signing and verification in a variety of environments and use cases.
+* [sigstore-policy-controller](https://github.com/sigstore/policy-controller) is the center of Sigstore-oriented verification flows in Kubernetes environments.
+
+# Roadmap
+
+## Client SDKs
+
+### Short Term (< 6 mo)
+
+* Releases:
```

Comment:
anything for sigstore-js here?

Reply:
@bdehamer Anything you wanted to call out? Any major releases in the next year?

Reply:
No major releases planned. I've got a long list of small features/improvements I'll be working on but nothing worth calling-out here.